### ISSUE TITLE
Fix disappearing node app

### ISSUE THREAD

**Issue by gz-c** [skycoin/skywire-testnet#82]
A user created a guide to fixing the disappearing node app bug,

https://medium.com/@CryptoRonny/a-solution-for-the-disappearing-node-app-key-in-skycoin-skyminer-with-autostart-scripts-on-7be5c1e465ca

This should be helpful for fixing the problem

**Comment by cypheron** [comment]
I don't think this is a bug. Nodes disappear because people have problems with automatic startup on boot on linux. A solution could be to include a autostart script or service script with the skywire installation.
